pages/signIn.tsx

```diff
@@ -14,19 +15,37 @@ const callSignInApi = async () => {
   console.log(await data.json())
 }
 
-
 function signUp(props) {
+  const [email, setEmail] = useState()
+  const [password, setPassword] = useState()
+  const [toolTip, setToolTip] = useState(false)
+  const [type, setType] = useState('')
```

Comment:
[request]  Add comments above the toolTip and type variable to explain what these are - just by looking at them I do not what they are there for

Reply:
done son